content/updates/2021-02/6-Rules-executed-4-times-a-day.md

```diff
@@ -0,0 +1,14 @@
+---
+pim_announcement_audience:
+- EE
+---
+
+# Rules execution 4 times a day
+::: meta-data type="New" features="Automation" available="mid-February" 
+
+We now execute rules more often, 4 times a day. This enhancement will drastically help you reduce your time to market. Indeed, your catalog is now more and more up-to-date with the latest computed values generated automatically by the Rules Engine.
```

Comment:
```suggestion
We now automatically execute rules 4 times a day. This enhancement will help you faster your enrichment process. Indeed, your catalog is now more and more up-to-date with the latest computed values generated automatically by the Rules Engine. Of course, if you have the permission to, you can also execute one or several rules manually from the Rules grid. 
```
Can we tell them when they are executed? (at what time?)

Reply:
Not sure about this. I'll ask.

Reply:
I'd remove the uppercase at Rules in "Rules grid". And remove the "s" too, as per Camille's recommendations.
```suggestion
We now automatically execute rules 4 times a day. This enhancement will help you faster your enrichment process. Indeed, your catalog is now more and more up-to-date with the latest computed values generated automatically by the Rules Engine. Of course, if you have the permission to, you can also execute one or several rules manually from the rule grid.
```

Reply:
No problem to cite the execution hours according to Momo. It's documented in the help center: https://help.akeneo.com/pim/serenity/articles/manage-your-rules.html#execute-rules
So I guess we can add some precision in our news too.

Reply:
```suggestion
We now automatically execute rules 4 times a day: at 5 am UTC, at 11 am UTC, at 5 pm UTC, and at 11 pm UTC. This enhancement will help you faster your enrichment process. Indeed, your catalog is now more and more up-to-date with the latest computed values generated automatically by the Rules Engine.

Note that, of course, rules are still regularly executed after a bulk action or after imports with rules, and also, if you are allowed to, you can still execute one or several rules manually from the rule grid.
```

Reply:
@LaureBrosseau are you ok with my last proposition?